src/Sarif.Converters/ClangAnalyzerConverter.cs

```diff
@@ -58,27 +57,7 @@ public override void Convert(Stream input, IResultLogWriter output, OptionallyEm
                     }
                 }
 
-                var tool = new Tool
-                {
-                    Name = "Clang"
-                };
-
-                var fileInfoFactory = new FileInfoFactory(MimeType.DetermineFromFileExtension, dataToInsert);
-                Dictionary<string, FileData> fileDictionary = fileInfoFactory.Create(results);
-
-
-                var run = new Run()
-                {
-                    Tool = tool
-                };
-
-                output.Initialize(run);
-
-                if (fileDictionary != null && fileDictionary.Count > 0) { output.WriteFiles(fileDictionary); }
-
-                output.OpenResults();
-                output.WriteResults(results);
-                output.CloseResults();
+                PersistResults(output, results, ToolFormat.ClangAnalyzer);
```

Comment:
>ClangAnalyzer [](start = 59, length = 13)

I don't think you should pass a `ToolFormat` to the `toolName` parameter. There are tools like FxCop and Fortify that can produce multiple formats. I think this confuses two concepts, the tool and its output format.